test/ethereum_test/src/transaction_test.cpp

```diff
@@ -113,14 +109,25 @@ void process_transaction(Transaction const &txn, nlohmann::json const &expected)
 }
 
 void process_transaction(
-    evmc_revision const rev, Transaction const &txn,
-    nlohmann::json const &expected)
+    std::variant<evmc_revision, monad_revision> const revision,
+    Transaction const &txn, nlohmann::json const &expected)
 {
-    MONAD_ASSERT(rev != EVMC_CONSTANTINOPLE);
-    SWITCH_EVM_TRAITS(process_transaction, txn, expected);
+    if (std::holds_alternative<evmc_revision>(revision)) {
+        auto const rev = std::get<evmc_revision>(revision);
+        MONAD_ASSERT(rev != EVMC_CONSTANTINOPLE);
+        SWITCH_EVM_TRAITS(process_transaction, txn, expected);
+    }
+    else {
+        auto const rev = std::get<monad_revision>(revision);
+        SWITCH_MONAD_TRAITS(process_transaction, txn, expected);
+    }
     MONAD_ASSERT(false);
```

Comment:
```suggestion
```
should be dead code now I think - compiler should be able to tell both of the branches return